jsonalchemy/json_encoder.py

```diff
@@ -0,0 +1,12 @@
+import json
+
+from jsonalchemy.dictstyle import JSONDict, JSONList
+
+class JSONAlchemyEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, JSONDict):
+            return {key: self.default(value) for key, value in dict(obj).iteritems()}
+        elif isinstance(obj, JSONList):
+            return [self.default(el) for el in obj]
+        else:
+            return obj
```

Comment:
This is not correct, because `json.JSONEncoder`'s default is to simply throw a `TypeError`: https://github.com/simplejson/simplejson/blob/master/simplejson/encoder.py#L228-L246.